src/shared/components/home/login-reset.tsx

```diff
@@ -0,0 +1,147 @@
+import { Component, linkEvent } from "inferno";
+import { GetSiteResponse } from "lemmy-js-client";
+import { i18n } from "../../i18next";
+import { UserService } from "../../services";
+import { HttpService } from "../../services/HttpService";
+import {
+  capitalizeFirstLetter,
+  isBrowser,
+  setIsoData,
+  toast,
+  validEmail,
+} from "../../utils";
+import { HtmlTags } from "../common/html-tags";
+import { Spinner } from "../common/icon";
+
+interface State {
+  form: {
+    email: string;
+  };
+  loading: boolean;
+  siteRes: GetSiteResponse;
+}
+
+export class LoginReset extends Component<any, State> {
+  private isoData = setIsoData(this.context);
+
+  state: State = {
+    form: {
+      email: "",
+    },
+    loading: false,
+    siteRes: this.isoData.site_res,
+  };
+
+  constructor(props: any, context: any) {
+    super(props, context);
+  }
+
+  componentDidMount() {
+    if (UserService.Instance.myUserInfo) {
+      this.context.router.history.push("/");
+    }
+  }
+
+  get documentTitle(): string {
+    return `${capitalizeFirstLetter(i18n.t("forgot_password"))} - ${
+      this.state.siteRes.site_view.site.name
+    }`;
+  }
+
+  get isLemmyMl(): boolean {
+    return isBrowser() && window.location.hostname == "lemmy.ml";
+  }
+
+  render() {
+    return (
+      <div className="container-lg">
+        <HtmlTags
+          title={this.documentTitle}
+          path={this.context.router.route.match.url}
+        />
+        <div className="row">
```

Comment:
If there's only going to be one row, why not make the parent div have a className of `container-lg row` and get rid of this element?

Reply:
Primarily to make it match up with `/login`. I've refactored this in my upcoming changes.